tracer.go

```diff
@@ -24,6 +24,8 @@ const (
 	QueryParametersKey = attribute.Key("pgx.query.parameters")
 	// BatchSizeKey represents the batch size.
 	BatchSizeKey = attribute.Key("pgx.batch.size")
+	// PrepareStmtNameKey represents the prepared statement name.
+	PrepareStmtNameKey = attribute.Key("pgx.prepare_stmt.name")
```

Comment:
not a blocker for this MR, but this feels like something that could be added to semconv: https://github.com/open-telemetry/semantic-conventions/blob/9790d8f54533840dd187d3ffa5c2546d9420d357/docs/database/sql.md